### PR TITLE
LLE : Extended NV2A color formats

### PR DESCRIPTION
**Pull request by PatrickvL**
This pull request completes the set of NV2A texture formats.

(Most additions still need verification, some need completion and/or conversion. But this is a starting point for that.)

### CODE REVIEW

**Comment by RadWolfie** [comment]
Also are you going to push these changes to XQEMU as well? I'm sure they'll want this.

File: src/devices/video/EmuNV2A_PGRAPH.cpp [hunk 1]
```diff
@@ -182,16 +182,17 @@ static const ColorFormatInfo kelvin_color_format_map[256] = {
...
-		{},
-	//0x17 [?] =
-		{},
-	//0x18 [?] =
-		{},
+	//0x14 [NV097_SET_TEXTURE_FORMAT_COLOR_LU_IMAGE_SY8] =
+        {1, true, GL_R8, GL_RED, GL_BYTE, /* FIXME: This might be signed */
+         {GL_RED, GL_RED, GL_RED, GL_ONE}}, // TODO : Verify
+	//0x15 [NV097_SET_TEXTURE_FORMAT_COLOR_LU_IMAGE_X7SY9] =
+		{2, true}, // TODO : Complete
```
[the remark this answers is not in the document]
Comment (PatrickvL) [reply src/devices/video/EmuNV2A_PGRAPH.cpp]: It's meant as : "to do : complete (this declaration)"

[the remark this answers is not in the document]
Comment (RadWolfie) [reply src/devices/video/EmuNV2A_PGRAPH.cpp]: Then fill in the blanks. One, "Complete", word for TODO list makes no sense at all. 😋

[the remark this answers is not in the document]
Comment (PatrickvL) [reply src/devices/video/EmuNV2A_PGRAPH.cpp]: I'll see what I can do tomorrow

**Comment by JayFoxRox** [comment]
I think had originally left a comment on this regarding useless texture conversion in certain cases which are also supported by GL (R5G6B5 for example).

Some other formats are probably incorrect.

However, generally I'd appreciate if some of these changes were also done upstream if they have been confirmed to work, and improve code readability.

**Pull request merged**